src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/submissionApiDelegateImpl/GetPackageInformationTest.java

```diff
@@ -0,0 +1,105 @@
+package ca.bc.gov.open.jag.efilingapi.submission.submissionApiDelegateImpl;
+
+import ca.bc.gov.open.jag.efilingapi.TestHelpers;
+import ca.bc.gov.open.jag.efilingapi.api.model.GetPacakageInformationResponse;
+import ca.bc.gov.open.jag.efilingapi.api.model.GetSubmissionResponse;
+import ca.bc.gov.open.jag.efilingapi.api.model.ParentApplication;
+import ca.bc.gov.open.jag.efilingapi.config.NavigationProperties;
+import ca.bc.gov.open.jag.efilingapi.submission.SubmissionApiDelegateImpl;
+import ca.bc.gov.open.jag.efilingapi.submission.mappers.GenerateUrlResponseMapper;
+import ca.bc.gov.open.jag.efilingapi.submission.models.Submission;
+import ca.bc.gov.open.jag.efilingapi.submission.service.SubmissionService;
+import ca.bc.gov.open.jag.efilingapi.submission.service.SubmissionStore;
+import ca.bc.gov.open.jag.efilingcommons.model.AccountDetails;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@DisplayName("SubmissionApiDelegateImpl test suite")
+public class GetPackageInformationTest {
+
+    private static final String APPL_TYPE = "ApplType";
+    private static final String KELWONA_LAW_COURTS = "Kelwona law courts";
```

Comment:
I know this is a test but any specific reason why kelowna?

Reply:
I pulled it from the examples. I used the example where provided.